src/api/schemas.ts

```diff
@@ -371,6 +387,42 @@ export const Brc20BalanceResponseSchema = Type.Object({
 });
 export type Brc20BalanceResponse = Static<typeof Brc20BalanceResponseSchema>;
 
+export const Brc20ActivityResponseSchema = Type.Object({
+  ticker: Type.String({ examples: ['PEPE'] }),
+  operation: Type.String({ examples: ['deploy', 'mint', 'prepare_transfer', 'transfer'] }),
+  inscription_id: Type.String({
+    examples: ['1463d48e9248159084929294f64bda04487503d30ce7ab58365df1dc6fd58218i0'],
+  }),
+  block_height: Type.Integer({ examples: [778921] }),
+  block_hash: Type.String({
+    examples: ['0000000000000000000452773967cdd62297137cdaf79950c5e8bb0c62075133'],
+  }),
+  tx_id: Type.String({
+    examples: ['1463d48e9248159084929294f64bda04487503d30ce7ab58365df1dc6fd58218'],
+  }),
+  address: Type.String({
+    examples: ['bc1pvwh2dl6h388x65rqq47qjzdmsqgkatpt4hye6daf7yxvl0z3xjgq247aq8'],
+  }),
+  deploy_max: Type.String({ examples: ['21000000'] }),
+  deploy_limit: Nullable(Type.String({ examples: ['100000'] })),
+  deploy_decimals: Nullable(Type.Integer({ examples: [18] })),
+  mint_amount: Nullable(Type.String({ examples: ['1000000'] })),
+  transfer_amount: Nullable(Type.String({ examples: ['1000000'] })),
+  transfer_from: Nullable(
+    Type.String({
+      examples: ['bc1pvwh2dl6h388x65rqq47qjzdmsqgkatpt4hye6daf7yxvl0z3xjgq247aq8'],
+    })
+  ),
+  transfer_to: Nullable(
+    Type.String({
+      examples: ['bc1pvwh2dl6h388x65rqq47qjzdmsqgkatpt4hye6daf7yxvl0z3xjgq247aq8'],
+    })
+  ),
```

Comment:
Instead of nullables (which should be optionals really) a better idea would be to send a nested object with separate schemas depending on the event type